Keas.Mvc/ClientApp/components/Equipment/EquipmentContainer.tsx

```diff
@@ -94,13 +100,56 @@ export default class EquipmentContainer extends React.Component<IProps, IState>
             onEdit={this._editEquipment}
             closeModal={this._closeModals}
             modal={activeAsset && (action === "edit")}
+            tags={this.state.tags}
             commonAttributeKeys={this.state.commonAttributeKeys}
             />
         </div>
       </div>
     );
   }
 
+  private _renderTableOrList = () => {
+    if(!!this.props.person)
+    {
+      return(
+      <div>
+        <EquipmentList
+          equipment={this.state.equipment}
+          onRevoke={this._revokeEquipment}
+          onAdd={this._openAssignModal}
+          showDetails={this._openDetailsModal}
+          onEdit={this._openEditModal}
+        />
+    </div>);
+    }
+    else
+    {
+      let filteredEquipment = this.state.equipment;
+      if(!!this.state.tagFilters && this.state.tagFilters.length > 0)
```

Comment:
@laholstege FYI I don't think the `!!` is doing anything here -- an array is truthy if it exists and shouldn't need the coalescence that a string would require.  However it doesn't hurt anything either, just makes it a very small bit harder to read.